internal/component/faro/receiver/exporters_test.go

```diff
@@ -12,10 +12,10 @@ import (
 )
 
 var metricNames = []string{
-	"logs_total",
-	"measurements_total",
-	"exceptions_total",
-	"events_total",
+	"faro_receiver_logs_total",
+	"faro_receiver_measurements_total",
+	"faro_receiver_exceptions_total",
+	"faro_receiver_events_total",
```

Comment:
Tests in this PR are updated due to a breaking change in `prometheus/client_golang`, nothing related to `mysqld_exporter` itself.